packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -236,6 +236,13 @@ export interface TableOptions extends SchemaOptions {
    * @default false
    */
   readonly contributorInsightsEnabled?: boolean;
+
+  /**
+   * ARN of the Kinesis Data Stream for the table.
+   *
+   * @default - no Kinesis Data Stream
+   */
+  readonly kinesisStreamArn?: string;
```

Comment:
This should be `readonly kinesisStream: kinesis.IStream`.

Reply:
Actually, please also move this to `TableProps` instead of `TableOptions`.

Reply:
> Actually, please also move this to `TableProps` instead of `TableOptions`.

Happy to move this. Interested to hear how you decide what belongs in `TableOptions` and what goes to `TableProps`?